manim/scene/scene_file_writer.py

```diff
@@ -40,6 +40,55 @@
     from manim.renderer.opengl_renderer import OpenGLRenderer
 
 
+def to_av_frame_rate(fps):
+    epsilon1 = 1e-4
+    epsilon2 = 0.02
+
+    if isinstance(fps, int):
+        (num, denom) = (fps, 1)
+    elif abs(fps - round(fps)) < epsilon1:
+        (num, denom) = (round(fps), 1)
+    else:
+        denom = 1001
+        num = round(fps * denom / 1000) * 1000
+        if abs(fps - num / denom) >= epsilon2:
+            raise ValueError("invalid frame rate")
+
+    return av.utils.Fraction(num, denom)
```

Comment:
Do you know if access to the class is public? I couldn't find it on their docs - so it might be better to just use the stdlib `fractions.Fraction` directly?